benchmarks/spack/csd3-centos7/cascadelake/spack.yaml

```diff
@@ -186,14 +332,22 @@ spack:
         prefix: /usr/local/software/hdf5/1.10.3
       - spec: hdf5@1.10.5%gcc@9.3.0+mpi^openmpi
         prefix: /usr/local/Cluster-Apps/hdf5/openmpi/gcc/9.2/1.10.5
-      - spec: hdf5@1.10.5%intel+mpi^intel-mpi
+      - spec: hdf5@1.10.5 +mpi^intel-mpi
```

Comment:
I presume the point of `%intel` was not to allow using this with other libraries built with gcc, I may have had troubles trying this combination

Reply:
Yeah, for some reason Spack refused to used this external package (when using the `intel` compiler and `intel-mpi`) until I made this change. I didn't really understand it since it didn't seem to be a problem for any other external packages with compilers in their specs :shrug: